src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -63,6 +63,20 @@ namespace {{name}} {
 } // namespace {{name}}
 {{/zcl_structs}}
 
+{{#zcl_commands}}
+{{#first}}
+namespace Commands {
```

Comment:
If we're doing this extra namespace here for Commands, shouldn't we do the same for Structs?

Reply:
We could, but we don't have to, since we are namespacing all the attributes and commands... unless we end up with a struct named "Attributes" or "Commands", of course.